x/monitoringc/module_simulation.go

```diff
@@ -18,7 +18,6 @@ import (
 // avoid unused import issue
 var (
 	_ = sample.AccAddress
-	_ = monitoringcsimulation.FindAccount
 	_ = simappparams.StakePerAccount
 	_ = simulation.MsgEntryKind
 	_ = baseapp.Paramspace
```

Comment:
Why we need those?
@Pantani do you have some clue?

Reply:
I'm not exactly sure actually.  Seems like I can remove all of them and can build / simulate everything fine.

Reply:
See #612